contactform.php

```diff
@@ -331,7 +331,7 @@ public function getTemplateVarContact()
             $contacts[$one_contact['id_contact']] = $one_contact;
         }
 
-        if (isset($this->customer_thread['id_contact'])) {
+        if (isset($this->customer_thread['id_contact']) && $this->customer_thread['id_contact'] !== 0) {
```

Comment:
```suggestion
        if (!empty($this->customer_thread['id_contact'])) {
```
The same only shorter 😉